Week1/homework/js-excercises/programming-jokes.js

```diff
@@ -0,0 +1,44 @@
+// I assumed that fetching the same issue at all times is kind of boring, so I decided to randomize the issue fetched.
+const randomIssue = () => Math.floor(Math.random() * 2301) + 1;
```

Comment:
why `+ 1` ?
can this work?
```suggestion
const randomIssue = () => Math.ceil(Math.random() * 2301);
```

Reply:
Hi, Meno-101, Thanks for your feedback. The random value should be between 1 and 2301, both inclusive. The formula should be as I wrote. I tried yours, it excludes the min(1), and adds a one to the max, so it becomes between 2 and 2302, inclusive.
I used this function getRandomIntInclusive(min, max) {
  
  return Math.floor(Math.random() * (max - min + 1)) + min; //The maximum is inclusive and the minimum is inclusive 
}

Reply:
Hi, I'm truly sorry for the late reply, but are you sure about what you have thought?

try this:
```
for (let i = 0; i < 100; i++) {
  console.log(Math.ceil(Math.random() * 2))
}
```
and tell me what possibilities you get? if you get 2 and 1 only then I'm correct 😎 if not then you are correct 🤓 and always every day is school day no shame of learning. 🙂 

the `getRandomIntInclusive` is correct, but I think it's a bit complicated and redundant on this exercise.

Aaand we eventually will have same values.

Reply:
Hi, thanks for taking the time to clarify this. You are right. I made a mistake when I tested it, changing the floor to ceil, but without removing the last +1. Thank You again, it is simpler and easier to remember.